lib/UtapiClient.js

```diff
@@ -580,13 +582,40 @@ class UtapiClient {
                 // empty.
                 actionCounter = Number.isNaN(actionCounter)
                     || actionCounter < 0 ? 1 : actionCounter;
+
+                if (params.byteLength) {
+                    actionCounter -= 1;
```

Comment:
I am not sure I understand why we decrement this counter, would you mind clarifying